src/Diagnostics/variables.jl

```diff
@@ -137,9 +137,19 @@ function setup_variables()
         var_attrib(
             "kg kg^-1",
             "water vapor specific humidity",
-            "specific humidity",
+            "water vapor specific humidity",
```

Comment:
I think it's specific_humidity in cf standard names (sorry I forgot _ in the original code). I do think water_vapor_specific_humidity is more accurate - up to you.

Reply:
Sounds good. We should stick to the CF convention :)